hybrasyl/Utility.cs

```diff
@@ -192,12 +192,6 @@ static class Constants
         public const int IDLE_TIME = 60;
         public const int IDLE_CHECK = 10;
 
-        // Shutdown password
-        // This is a dirty hack until we have better role / auth support
-
-        public const string ShutdownPassword = "batterystaple8!";

```

Comment:
RIP batterystaple8

Reply:
RIP 2014-2020
press f in chat to show respects